defaults/main.yml

```diff
@@ -37,12 +37,19 @@ elasticsearch_yum_repo_baseurl: "https://artifacts.elastic.co/packages/{{ elasti
 
 ###########
 ##  Plugins
-elasticsearch_update_plugins:  false
-elasticsearch_plugins:         []
+elasticsearch_update_plugins:  true
+elasticsearch_plugins:         [x-pack]
 elasticsearch_plugins_install: "{{ elasticsearch_plugins|default([]) }}"
 elasticsearch_plugins_remove:  []
 elasticsearch_plugin_cmd:      "{{ elasticsearch_home_dir }}/bin/elasticsearch-plugin"
 
+### Xpack related features ###
+elasticsearch_xpack_security: false
+elasticsearch_xpack_ml: false
+elasticsearch_xpack_watcher: false
+elasticsearch_xpack_monitoring: false
+
```

Comment:
missing `xpack.reporting.enabled: true`? Or is this set somewhere else?

Reply:
Xpack Reporting is a feature only for Kibana and not Elasticsearch. Elasticsearch throws an error if try to enable reporting.

https://www.elastic.co/guide/en/x-pack/6.2/xpack-settings.html